canarytokens/azure_css.py

```diff
@@ -10,6 +10,22 @@
 
 BearerToken = str
 
+EntraTokenErrorAccessDenied = "access_denied"
+
+ENTRA_BASE_REDIRECT_URL = "/nest/entra/{status}"
+
+ENTRA_STATUS_HAS_CUSTOM_CSS_ALREADY = "has_custom_css_already"
+ENTRA_STATUS_ERROR = "error"
+ENTRA_STATUS_SUCCESS = "success"
+ENTRA_STATUS_NO_ADMIN_CONSENT = "no_admin_consent"
```

Comment:
Can we use an enum for these?

Something like:
```suggestion
class EntraStatus(enum.Enum):
    HAS_CUSTOM_CSS_ALREADY = "has_custom_css_already"
    ERROR = "error"
    SUCCESS = "success"
    NO_ADMIN_CONSENT = "no_admin_consent"
    ERROR_ACCESS_DENIED = "access_denied"

ENTRA_BASE_REDIRECT_URL = "/nest/entra/{status}"
```

Then we just import that one thing on the other side and check `EntraStatus.<whatever>` instead

Reply:
the python version we use doesn't support StrEnum, that was the plan originally. enum.StrEnum. so without that we'd have to do `EntraStatus.<whatever>.value` or:

```
class EntraStatus(str, enum.Enum):
     HAS_CUSTOM_CSS_ALREADY = "has_custom_css_already"
     ERROR = "error"
     SUCCESS = "success"
     NO_ADMIN_CONSENT = "no_admin_consent"
     ERROR_ACCESS_DENIED = "access_denied"

     def __str__(self):
         return str(self.value)```